src/pages/StoreDetail/StoreDetail.tsx

```
@@ -67,29 +34,53 @@ const StoreDetail = () => {
+            pagination
+            navigation={{
+              prevEl: ".prevImg",
+              nextEl: ".nextImg",
+            }}
+            onSwiper={(e) => {
+              setSwiper(e);
+            }}
+          >
+            {imgSlide.map((slide, index) => (
+              <SwiperSlide>
+                <img key={index} src={slide} alt={`가게 이미지 ${index + 1}`} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)